src/Adapter/Platform/AbstractPlatform.php

```diff
@@ -1,9 +1,11 @@
 <?php
+
```

Comment:
Are you reviewing old commits by accident? I think this is before you told me not to run outdated CS fixer in another PR. I redid all my other PRs same evening, including this one.